src/main/resources/application-local.yml

```
@@ -12,11 +12,41 @@ spring:
+        provider:
+          kakao:
+            authorization-uri: https://kauth.kakao.com/oauth/authorize
+            token-uri: https://kauth.kakao.com/oauth/token
+            user-info-uri: https://kapi.kakao.com/v2/user/me
+            user-name-attribute: id
+
+  jwt:
+    header: Authorization
+    secret: ${JWT_SECRET}
+    access-token-validity-in-seconds: 3600
+    refresh-token-validity-in-seconds: 2592000
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)